apps/web/vitest.config.setup.ts

```diff
@@ -0,0 +1,105 @@
+import { createI18n } from 'vue-i18n';
+import { config } from '@vue/test-utils';
+
+const FindTestIdPlugin = (wrapper: any) => {
+  function findByTestId(testId: string) {
+    return wrapper.find(`[data-testid='${testId}']`);
+  }
+
+  return {
+    findByTestId,
+  };
+};
+
+const GetTestIdPlugin = (wrapper: any) => {
+  function getByTestId(testId: string) {
+    return wrapper.get(`[data-testid='${testId}']`);
+  }
+
+  return {
+    getByTestId,
+  };
+};
+
+config.plugins.VueWrapper.install(FindTestIdPlugin);
+config.plugins.VueWrapper.install(GetTestIdPlugin);
+
+const i18n = createI18n({
+  legacy: false,
+  fallbackWarn: false,
+  missing: (_locale, key) => key,
+});
+config.global.plugins = [i18n];
+config.global.mocks = {
+  $t: (key: string) => key,
+  $d: (date: Date) => date.toLocaleDateString(),
+};
+config.global.stubs = {
+  RouterLink: true,
+  NuxtImg: true,
+  'i18n-t': true,
+};
+
+import { mockComponent, mockNuxtImport } from '@nuxt/test-utils/runtime';
+import { CategoryTreeMock } from './__tests__/fixtures/category-tree.mock';
+import { ShippingCountriesMock } from './__tests__/fixtures/shipping-countries.mock';
```

Comment:
**question**: Is it proper that we spread imports throughout the file?

Reply:
no I will move them